src/pageManager.js

```diff
@@ -488,19 +488,18 @@ export async function initialize() {
     // Register background script event handlers
 
     // If a tab's audible state changed, send webScience.pageManager.pageAudioUpdate
-    browser.tabs.onUpdated.addListener((tabId, changeInfo) => {
+    browser.tabs.onUpdated.addListener((tabId, changeInfo, extraParameters) => {
         if(!initialized) {
             return;
         }
 
-        messaging.sendMessageToTab(tabId, {
-            type: "webScience.pageManager.pageAudioUpdate",
-            pageHasAudio: changeInfo.audible,
-            timeStamp: timing.now()
-        });
-    }, {
-        urls: [ "http://*/*", "https://*/*" ],
```

Comment:
This filter is unsupported in Chrome, and it's not handled by the polyfill.

Reply:
Should we AND the condition here? audible && Firefox?

Reply:
Well we can still do it, but we need to handle it inside the callback vs. setting a filter on the `addListener` call. I just cleaned this up a little and corrected the way it works, it should be cross-browser now.